common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -140,6 +138,16 @@ def edit_notes(self, components, offset=0):
                 note.text = "TEST TEXT {}".format(index)
                 index += 1
 
+    def edit_tags_in_notes(self, components, tags):
+        self.assertGreater(len(components), 0)
```

Comment:
It's not causing a quality failure, it's consistent with the other similar helper methods in this class, and I don't see what value a doc string would provide given the descriptive name.